build.sbt

```diff
@@ -140,31 +155,45 @@ val root = (project in file("."))
   .aggregate(
     text.jvm,
     text.js,
+    text.native,
     csv.jvm,
     csv.js,
+    csv.native,
     csvGeneric.jvm,
     csvGeneric.js,
+    csvGeneric.native,
     json.jvm,
     json.js,
+    json.native,
     jsonCirce.jvm,
     jsonCirce.js,
+    jsonCirce.native,
     jsonDiffson.jvm,
     jsonDiffson.js,
+    jsonDiffson.native,
     jsonInterpolators.jvm,
     jsonInterpolators.js,
+    jsonInterpolators.native,
+    jsonPlay.jvm,
+    jsonPlay.js,
```

Comment:
The fact they were missing didn't seem to bother anyone, the only published version was 1.3.0: https://mvnrepository.com/artifact/org.gnieh/fs2-data-json-play

Trying to fix this now within this PR by releasing full Diffson support.

I'm in favour of dropping support in v2, that library seems rather dead and the code we maintain is so small that it could easily be inlined in users' projects.